sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  Body,
+  Controller,
+  Post,
+  UnprocessableEntityException,
+} from "@nestjs/common";
+import { InstitutionService, UserService } from "../../services";
+import { CreateInstitutionDto } from "./models/institution.dto";
+import { UserToken } from "../../auth/decorators/userToken.decorator";
+import { IUserToken } from "../../auth/userToken.interface";
+import BaseController from "../BaseController";
+
+@Controller("institution")
+export class InstitutionController extends BaseController {
+  constructor(
+    private readonly userService: UserService,
+    private readonly institutionService: InstitutionService,
+  ) {}
```

Comment:
after removing super call, get this error - 

src/route-controllers/institution/institution.controller.ts:15:3 - error TS2377: Constructors for derived classes must contain a 'super' call.

15   constructor(
     ~~~~~~~~~~~~
16     private readonly userService: UserService,
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
17     private readonly institutionService: InstitutionService,
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
18   ) {}
   ~~~~~~

Found 1 error(s).

Reply:
@andrewsignori-aot @fwpushan

Reply:
If you base controller super is required ...